src/background.js

```diff
@@ -7,3 +7,14 @@ chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
         chrome.action.setPopup({ tabId, popup: "popup_disabled.html" });
     };
 });
+
```

Comment:
On install, it should open this usage link so the user has instructions on how to use it from the start. However, if tested right now without merging, this wouldn't work, as the main branch would not have the usage.md. If you want to see how it would look, look at the commented link.